### PR TITLE
Updated documentation around pm2 to discuss global pm2 options

### PR DESCRIPTION
**Pull request by zpeters**
started on a draft of additional pm2 direction for review.

Still working on the global pm2 switch.  Will add additional detail when that is ready.

### CODE REVIEW

**Comment by jtoar** [comment]
@cannikin May have request a review from you too soon. @zpeters let us know when it's ready—thanks for taking this on!

**Comment by zpeters** [comment]
Added notes around the global pm2 options.  I don't know if im fully happy with the wording.  Open to suggestions!

File: docs/docs/deploy/baremetal.md [hunk 1]
```diff
@@ -55,11 +55,16 @@ This will create a couple of files and add a dependency or two to your `package.
...
 
-If you see an error from `gyp` you may need to add some additional dependencies before `yarn install` will be able to complete. See the README for `node-type` for more info: https://github.com/nodejs/node-gyp#installation
+Since pm2 is not added by default, you will need to take some additional action to include this into your project.  For 'local' mode (the default) you will run the command `yarn add pm2` to add this as a project dependency in your `packages.json`.  From here everything else is automated.
+
+If you choose to run pm2 'globally' you would need to follow the pm2 projects instructions for setting this up.  https://pm2.keymetrics.io/docs/usage/quick-start/.  Typically this is `npm install pm2@latest -g` as root
```
Comment (cannikin) [inline docs/docs/deploy/baremetal.md:67]:
```suggestion
If you choose to run pm2 'globally' you would need to follow the pm2 project's instructions for setting this up.  https://pm2.keymetrics.io/docs/usage/quick-start/.  Typically this is `npm install pm2@latest -g` as root.
```

**Review by cannikin — CHANGES_REQUESTED** [review]

**Comment by cannikin** [comment]
We can keep the Self-Hosting doc for now, but we should probably deprecate that doc and refer people to the Docs > Deployment > Baremetal docs going forward.

**Comment by cannikin** [comment]
I've done a bunch of updates to this doc in https://github.com/redwoodjs/redwood/pull/5282 which covers the discussion of a global install, thanks!

**Pull request closed without merging**